ci_scripts/build_macos.sh

```diff
@@ -124,7 +134,7 @@ popd
 #
 
 echo "Configure KTX-Software (iOS)"
-cmake -GXcode -Bbuild-ios -DISA_NEON=ON -DCMAKE_SYSTEM_NAME=iOS -DKTX_FEATURE_LOADTEST_APPS=ON -DKTX_FEATURE_DOC=OFF
+cmake -GXcode -Bbuild-ios -DISA_NEON=ON -DCMAKE_SYSTEM_NAME=iOS -DKTX_FEATURE_LOADTEST_APPS=ON -DKTX_FEATURE_DOC=OFF -DKTX_FEATURE_JNI=ON
```

Comment:
I didn't notice this before. Is Java available for iOS? I thought the same license restrictions that killed Flash would prevent Java from being run.

If it is not, then `KTX_FEATURE_JNI` should be OFF for iOS and it should be an error to try to set it.

Reply:
https://openjdk.java.net/projects/mobile/ios.html

Java is available for iOS because of AOT technology.